packages/ckeditor5-image/theme/image.css

```diff
@@ -4,6 +4,13 @@
  */
 
 .ck-content {
+	& picture {
+		/* The image should never exceed the width of the parent container */
+		& img {
+			max-width: 100%;
+		}
+	}
```

Comment:
Is this part correct? I was surprised when I saw the `picture` here.

Reply:
It's correct. But actually I am not planning to add this CSS rule at all to be consistent with other inline images. But I'd keep the manual tests introduced in this ticket.